packages/runtime/test/integration/updateApp.ts

```diff
@@ -172,4 +171,144 @@ describe("Algorand Smart Contracts - Update Application", function () {
 		assert.deepEqual(runtime.getGlobalState(appID, "global-key"), undefined);
 		assert.deepEqual(runtime.getLocalState(appID, creator.address, "local-key"), undefined);
 	});
+
+	describe("Extra Pages", function () {
```

Comment:
please add details (change the string) to the name of this suite.

Reply:
Done